app/experimenter/experiments/constants/nimbus.py

```diff
@@ -215,6 +215,86 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_INFREQUENT_USER_URIS = NimbusTargetingConfig(
+    name="Infrequent user (uris)",
+    slug="infrequent_user_uris",
+    description="Between 1 and 6 URIs visited in the past 28 days",
```

Comment:
@irrationalagent This description is indeed misleading as it's actually counting 1-6 days (that have at least 1 URI visited). The data does contain the URI counts from a given day, but here we're looking at `userMonthlyActivity|length` which is the total length of the array where each array item is a pair `[uri count, date]`

Here's an example value of `userMonthlyActivity`:
```js
[
  [
    1038,
    "17-08-2021"
  ],
  [
    830,
    "18-08-2021"
  ],
  [
    140,
    "19-08-2021"
  ],
]
```

You can view your own data by:
1) toggling `browser.newtabpage.activity-stream.asrouter.devtoolsEnabled` from about:config
2) opening a new tab
3) click wrench icon top right corner
4) select Targeting on the left side
5) scroll down (or find in page) "userMonthlyActivity" or "mainPingSubmissions"

Reply:
Thanks for walking me through that. 

RE: the 60 day issue for  `mainPingSubmissions` - I wonder about the extent to which we really need to use this, if we already have `userMonthlyActivity`. 

In other words, based on my understanding of things, if the user has non-0 URIs for a given date, then they should have, if everything worked properly, submitted a main ping on that day as well. I get there are edge cases there where a user browses on a day but e.g. kills their machine before they send a ping. And we may want to know the extent to which things *do* operate this way.

That said, `userMonthlyActivity` represents the client's direct understanding of how many URIs were browsed on a day. That's what we're trying to *estimate* on the server side by counting URIs per (server) submission date. So, even though just relying solely on `userMonthlyActivity` will to some extent conflict with our server-side understanding, maybe it would be better because its closer to what we are *intending* to target on. 

And, once we have telemetry that rolls up `userMonthlyActivity` and sends it to us, we won't have to rely on server-based understandings anymore, anyway. 

So to be clear, assuming I'm not misunderstanding things here (big if), I think I'm advocating that we rely only on `userMonthlyActivity` for the segment targeting.